Sources/TSCBasic/WritableByteStream.swift

```diff
@@ -625,19 +648,24 @@ public final class BufferedOutputByteStream: _WritableByteStreamBase {
     override final func writeImpl(_ bytes: ArraySlice<UInt8>) {
         contents += bytes
     }
+
+    override final func closeImpl() throws {
+        // Do nothing. The protocol does not require to stop receiving writes, close only signals that resources could
+        // be released at this point should we need to.
+    }
 }
 
 /// Represents a stream which is backed to a file. Not for instantiating.
 public class FileOutputByteStream: _WritableByteStreamBase {
```

Comment:
Would this at some point be renamed `FileWritableByteStream` or something similar?

Reply:
potentially? not super concerned. The protocol is more important IMO